onlinejudge_verify/documentation/build.py

```diff
@@ -107,9 +107,9 @@ def ext(relative_path: pathlib.Path) -> Dict[str, Any]:
             'icon': _get_verification_status_icon(stat.verification_status),
         }
 
-    data['_extendedDependsOn'] = [ext(path) for path in stat.depends_on]
-    data['_extendedRequiredBy'] = [ext(path) for path in stat.required_by]
-    data['_extendedVerifiedWith'] = [ext(path) for path in stat.verified_with]
+    data['_extendedDependsOn'] = sorted([ext(path) for path in stat.depends_on], key=lambda x: x['path'])
+    data['_extendedRequiredBy'] = sorted([ext(path) for path in stat.required_by], key=lambda x: x['path'])
+    data['_extendedVerifiedWith'] = sorted([ext(path) for path in stat.verified_with], key=lambda x: x['path'])
```

Comment:
```suggestion
    data['_extendedDependsOn'] = [ext(path) for path in sorted(stat.depends_on, key=lambda x: x.path)]
    data['_extendedRequiredBy'] = [ext(path) for path in sorted(stat.required_by, key=lambda x: x.path)]
    data['_extendedVerifiedWith'] = [ext(path) for path in sorted(stat.verified_with, key=lambda x: x.path)]
```

nit: `sorted` は内側の方が読みやすい (すくなくとも型検査は効いてくれる) と思うのですがどうでしょう (たぶん好みの問題ですが…)

Reply:
特にこだわり、なし…（CI通ったらマージします